### ISSUE TITLE
Can only see some of the BTTV emotes.

### ISSUE THREAD

**Issue by Pyrrhomaniac** [night/BetterTTV#3140]
I have full access to the BTTV emotes menu, but can't see some of them as anything other than text in chat, i.e. PepeHands, OMEGALUL, POGGERS.

<img width="342" alt="screen shot 2018-06-08 at 12 28 10 am" src="https://user-images.githubusercontent.com/17808083/41142437-f048f4f2-6ab2-11e8-87c9-5e528fe8481a.png">

<img width="340" alt="screen shot 2018-06-08 at 12 30 48 am" src="https://user-images.githubusercontent.com/17808083/41142526-3e6eb626-6ab3-11e8-8df7-a906d0970058.png">

**Comment by night** [comment]
Are you sure "BetterTTV GIF Emotes" is turned on in your BetterTTV settings?

**Comment by Pyrrhomaniac** [comment]
Yes, they are turned on and I can see some of them in chat. I've included additional screenshots of my BTTV settings to see if that helps. I went through them all before, but couldn't find anything that could logically make sense of the situation.

<img width="799" alt="screen shot 2018-06-09 at 2 50 55 pm" src="https://user-images.githubusercontent.com/17808083/41196026-498db5dc-6bf5-11e8-8e58-125776f2d26d.png">
<img width="800" alt="screen shot 2018-06-09 at 2 52 11 pm" src="https://user-images.githubusercontent.com/17808083/41196023-49533556-6bf5-11e8-89dc-5739559e0af1.png">
<img width="801" alt="screen shot 2018-06-09 at 2 52 22 pm" src="https://user-images.githubusercontent.com/17808083/41196024-49682d80-6bf5-11e8-8b8f-5cd166838b12.png">
<img width="802" alt="screen shot 2018-06-09 at 2 52 33 pm" src="https://user-images.githubusercontent.com/17808083/41196025-497bb206-6bf5-11e8-8e83-08ca2b83fb1d.png">
